temporalcli/commands.gen.go

```diff
@@ -473,10 +475,35 @@ func NewTemporalActivityFailCommand(cctx *CommandContext, parent *TemporalActivi
 	return &s
 }
 
+type ActivityReferenceOptions struct {
+	ActivityId   string
+	ActivityType string
+}
+
+func (v *ActivityReferenceOptions) buildFlags(_ *CommandContext, f *pflag.FlagSet, activityType string, onlyWithoutQuery bool) {
```

Comment:
You cannot customize flag building in this way with our framework as it is today. You can make different option sets or, as we have done in other cases, accept that documentation is shared when option sets are shared.